gdp-new-hmi: Change name to gdp-hmi

chbae opened the pull request:
genivi-11-hmi that is previous binary name is only for GDP-11 and need
to change general naming. So I change the name to gdp-hmi.

Signed-off-by: Changhyeok Bae <changhyeok.bae@gmail.com>

zeenix commented on meta-genivi-dev/recipes-dev-hmi/genivi-dev-platform-hmi/gdp-new-hmi.bb:
```diff
@@ -1,40 +1,35 @@
+RDEPENDS_${PN} += "qtsvg"
 
-RDEPENDS_${PN} += "qtbase qtsvg"
-
```
Please put each logical change in it's own patch/commit with it's own commit log, no matter how small. This patch should only and only contain changes need to rename the binary (source files/recipe renames can & should also be part of it but then you need to remove 'binary' from commit log).

zeenix commented:
@chbae will you be working on this?

chbae commented:
I'll apply your review tonight. :)

chbae commented:
@zeenix 
DONE. Please review this.

zeenix commented on meta-genivi-dev/recipes-dev-hmi/genivi-dev-platform-hmi/gdp-new-hmi.bb:
```diff
@@ -1,40 +1,35 @@
+    mkdir -p ${D}/home/root/.config/systemd/user/default.target.wants/gdp-new-hmi.service
+    ln -sf ${libdir}/systemd/user/gdp-new-hmi.service ${D}/home/root/.config/systemd/user/default.target.wants/gdp-new-hmi.service
+    install -d ${D}${datadir}/applications/
+    install -m 0444 ${WORKDIR}/git/manifests/* ${D}${datadir}/applications/
```
Sorry to be very pedantic about this but here you're also changing the indentation, which is unrelated to the patch and hence needs to be done separately. I suggest breaking it out into a separate patch that goes before this one.

chbae committed:
gdp-new-hmi: Update to latest git master

This also involves update of binary name from genivi-11-hmi to gdp-hmi in
the service file.

Signed-off-by: Changhyeok Bae <changhyeok.bae@gmail.com>

chbae commented:
@zeenix 
Now I only change the naming.

zeenix commented on meta-genivi-dev/recipes-dev-hmi/genivi-dev-platform-hmi/gdp-new-hmi.bb:
```diff
@@ -1,5 +1,5 @@
 SRC_URI = "git://github.com/GENIVI/hmi-layout-gdp.git"
-SRCREV = "d0ca9d072012d8ac333537d9b922d9242e022efe"
+SRCREV = "79a4f69e7588a6efc4b4c95abec7794216588087"
```
Also changing the revision still. :)

chbae replied:
This change is for renaming. See https://github.com/GENIVI/hmi-layout-gdp/commit/79a4f69e7588a6efc4b4c95abec7794216588087

zeenix replied:
Ah I see now that you changed the name in HMI itself. In this case, the update of HMI here is the real change and renaming is just a side-effect, from the point of view of meta-genivi-dev. Hence I'd like the commit log to be changed.

chbae replied:
@zeenix I thought that current commit message is enough. Please write it what you want. Thanks.

zeenix replied:
@chbae It's not that it's not enough, it's that it's wrong. You're summarizing a side-effect, not the change. I can't be writing commit messages for everyone. :) Please try yourself first and if you fail, I'll help. Deal? :)

chbae replied:
@zeenix 

My commit is not about side effect and 79a4f69e7588a6efc4b4c95abec7794216588087 is also for changing naming. So 79a4f69e7588a6efc4b4c95abec7794216588087 in component repo and my recipe change are for naming changing.

I thought that you don't understand this change.

zeenix replied:
As I tried to explain above, from the **point of view of m-g-d** (which is what you are changing here), change of name is a side-effect. It's very similar to submodule updates in g-d-p repo and you can see from the log (and our new script) how we don't mention the actual changes in the shortlog but rather in the description.

Anyway, I'll modify the commit log for you.

chbae replied:
@zeenix Ok. thanks. However, I think that recipe and repo change is one package.
That's different from each reviewers. If you give your idea, I respect you.

zeenix commented on meta-genivi-dev/recipes-dev-hmi/genivi-dev-platform-hmi/gdp-new-hmi.bb:
```diff
@@ -22,7 +22,7 @@ SRC_URI_append_rcar-gen2 ="\
 FILES_${PN} += "\
     ${libdir}/* \
-    /opt/genivi-11-hmi/bin/genivi-11-hmi \
+    /opt/gdp-hmi/bin/gdp-hmi \
```
So is this the only change that was needed for rename?

chbae replied:
no. need to change SRCREV.
